### PR TITLE
feat: Add unit tests for entity\history classes

### PR DESCRIPTION
**Pull request by o-kopysov**
# Description

We have to cover more classes by unit tests to improve code coverage.
In this PR `entity\history` classes are covered by additional unit tests.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] Code cleanup/refactoring
- [ ] Documentation update
- [ ] This change requires a documentation update
- [ ] CI system update
- [X] Test Coverage update

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/test/java/com/lpvs/entity/history/HistoryPageEntityTest.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        HistoryPageEntity entity1 = new HistoryPageEntity(prPage1, count1);
+        List<LPVSPullRequest> pullRequests2 = new ArrayList<>();
+        pullRequests2.add(new LPVSPullRequest(3L, null, "Title 3", null, null, null, null, null, null, null));
+        Page<LPVSPullRequest> prPage2 = new PageImpl<>(pullRequests2);
+        Long count2 = 1L;
+        HistoryPageEntity entity2 = new HistoryPageEntity(prPage2, count2);
+        assertFalse(entity1.equals(entity2));
+        assertFalse(entity2.equals(entity1));
+    }
+}
```
Comment (m-rudyk) [inline src/test/java/com/lpvs/entity/history/HistoryPageEntityTest.java:60]: Please add empty line

File: src/test/java/com/lpvs/entity/history/LPVSHistoryTest.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    public void testInequality() {
+        // Test inequality when objects are not the same
+        LPVSHistory history1 = new LPVSHistory("2023-10-12", "example/repository", 1L,
+            "https://example.com/pull/1", "Success", "john.doe", "pull/1", true);
+        LPVSHistory history2 = new LPVSHistory("2023-10-13", "another/repository", 2L,
+            "https://example.com/pull/2", "Failure", "jane.doe", "pull/2", false);
+        assertFalse(history1.equals(history2));
+        assertFalse(history2.equals(history1));
+    }
+}
```
Comment (m-rudyk) [inline src/test/java/com/lpvs/entity/history/LPVSHistoryTest.java:57]: same here

**Review by m-rudyk — APPROVED** [review]
Thanks for increasing coverage. Approved

**Review by tiokim — APPROVED** [review]
LGTM

**Pull request merged**